src/BUILD

```diff
@@ -5,12 +5,11 @@
 grapl_python_distribution(
     name = "graplinc_grapl_api",
     dependencies = [
-        "src/proto/graplinc/grapl/api",
-        "src/python/graplinc/graplinc/grapl/api"
+        "src/proto:graplinc",
```

Comment:
yes, this technically includes more than we need to, and I could do a proto library that's just the api. Not convinced it's worth it.

Reply:
Yes, I agree. I think all our protobuf stubs should be one "block", and ultimately they should be wrapped in a library. This library (`python-proto`, and its counterpart `rust-proto`) should be the sole point-of-contact for code which needs to serialize or deserialize protocol buffers. "Internal" code should not depend directly on the protobuf stubs.

This is one of the changes I'm attempting to introduce in https://github.com/grapl-security/grapl/pull/1121